optados/make.system

```diff
@@ -2,7 +2,7 @@
 SYSTEM := gfortran
 
 # fast / debug
-BUILD := fast 
+BUILD := debug 
```

Comment:
fast should be the default for a distribution.
Do we need to have a special makefile for Github actions (i.e. setting debug)

Reply:
When I get a chance, I'll merge this PR into my fork and try it out.

Reply:
Not necessary. I've reverted to : fast. It was unnecessary.